File: Hackerman.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c,i,p=0;
        scanf("%d%d",&a,&b);
        c=a+b;
        for(i=1;i<=c;i++)
        {
            if(c%i==0)
            p++;
        }
        if(p==2)
        printf("Alice\n");
        else
        printf("Bob\n");
    }
}

File: Hackerman.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int n=a+b,c=0;
            for(int i=1;i<=n;i++)
            {
                if(n%i==0)
                c++;
            }
            if(c==2)
            System.out.println("Alice");
            else
            System.out.println("Bob");
        }
    }
}